bundler/helpers/v2/spec/functions/version_resolver_spec.rb

```diff
@@ -29,6 +29,15 @@
 
   let(:rubygems_url) { "https://index.rubygems.org/api/v1/" }
   let(:old_index_url) { rubygems_url + "dependencies" }
+  let(:gemfury_url) { "https://repo.fury.io/greysteil/" }
+
+  before do
+    stub_request(:get, "https://rubygems.org/quick/Marshal.4.8/statesman-1.2.1.gemspec.rz").
+      to_return(status: 200, body: fixture("rubygems_responses", "statesman-1.2.1.gemspec.rz"))
+
+    stub_request(:get, %r{quick/Marshal.4.8/business-.*.gemspec.rz}).
+      to_return(status: 200, body: fixture("rubygems_responses", "business-1.0.0.gemspec.rz"))
```

Comment:
Same here, I might wonder in a future Bundler update why these had to change here and whether I should be updating them, too. I'd benefit from a quick comment re: why 🙂

Reply:
Same thing here, bundler internally now makes these calls, I didn't try to figure out why it does this because there's not much we can do about it anyway, I just added the requests being made to the list of stubbed requests